components/operator/internal/controllers/stack/testdata/monopod-disabled-one-service/results/configmaps--v1/search-benthos-streams.yaml

```diff
@@ -299,50 +299,70 @@ data:
 
         output:
           resource: elasticsearch
-    payments_ingestion.yaml: |
+    payments_deletion.yaml: |
         input:
           event_bus:
             topic: payments
-            consumer_group: search
+            consumer_group: search-payments-resets
 
         pipeline:
           processors:
           - switch_event_type:
               events:
-              - label: SAVED_PAYMENT
+              - label: CONNECTOR_RESET
                 processors:
                 - bloblang: |
                     root = {
-                        "data": this.payload,
-                        "indexed": {
-                            "provider": this.payload.provider,
-                            "reference": this.payload.reference,
-                            "scheme": this.payload.scheme,
-                            "type": this.payload.type,
-                            "status": this.payload.status,
-                            "id": this.payload.id,
-                            "initialAmount": this.payload.initialAmount,
-                            "createdAt": this.payload.createdAt
-                        },
-                        "kind": "PAYMENT",
-                        "when": this.date
+                      "query": {
+                        "bool": {
+                          "must": [
+                            {
+                              "bool": {
```

Comment:
What is the behavior if matching against multiple value of "kind" ?

Reply:
Yes, must = AND and should = OR, so in this case, everything works as expected !